assets/sass/_content-post.scss

```diff
@@ -17,6 +17,17 @@
     text-transform: none;
   }
 
+  a.external::after {
```

Comment:
Much nicer way of doing this that doesn't require an additional class: https://davidwalsh.name/external-links-css